### PR TITLE
files are added

### PR DESCRIPTION
**Pull request by vighnesh1001**

### CODE REVIEW

File: utils/trainning.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            masks = masks.to(device)
+            optimizer.zero_grad()
+            
+            outputs = model(images)
+            loss_value = loss_fn(outputs, masks)
+            loss_value.backward()
+            optimizer.step()
+            
+            
+            if scheduler is not None:
```
Comment (gauthamk02) [inline utils/trainning.py:32]: check if this is the correct usage for onecycleLR

File: main.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        transforms.Resize((256, 256)),  
+        transforms.ToTensor(),
+        transforms.RandomHorizontalFlip(p=0.5),
+        transforms.RandomVerticalFlip(p=0.5),
+        transforms.RandomRotation(degrees=90),
+        transforms.ElasticTransform(alpha=50.0, sigma=5.0),
+        transforms.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
+    ])
+
+    mask_transforms = transforms.Compose([
```
Comment (gauthamk02) [inline main.py:36]: same transform should be applied to mask too
